src/Symfony/Component/Intl/Tests/NumberFormatter/AbstractNumberFormatterTest.php

```diff
@@ -789,13 +780,24 @@ public function parseTypeDoubleProvider()
         );
     }
 
-    public function testParseTypeCurrency()
+    private function handleExpectedWarningException()
```

Comment:
That was quit some work... 😄 Now it should be okay. On to the next one I would say...